src/styles/components/form-elements/styles.less

```diff
@@ -35,9 +41,20 @@
     position: absolute;
   }
 
-  .form-remove {
+  .form-group-container-action-button {
```

Comment:
It's a bit funky to read but it's actually used inside of the `FormGroupContainer` component, so I do think it makes sense... do you still think it should be changed?